concordium-std/src/types.rs

```diff
@@ -954,6 +954,18 @@ impl<S> StateBuilder<S> {
     pub fn into_inner(self) -> S { self.state_api }
 }
 
+/// A struct for which HasCryptoUtils is implemented via the crypto host
+/// functions.
+#[doc(hidden)]
+pub struct ExternCryptoUtils;
+
+pub type PublicKeyEd25519 = [u8; 32];
+pub type PublicKeyEcdsaSecp256k1 = [u8; 33];
+pub type SignatureEd25519 = [u8; 64];
+pub type SignatureEcdsaSecp256k1 = [u8; 64];
+pub type MessageHashEcdsaSecp256k1 = [u8; 32];
```

Comment:
I don't think we need to have this type as a separate type. I think it's OK to just use [u8; 32] in the verification function, since that is really what it needs.